ql/src/test/org/apache/hadoop/hive/ql/exec/repl/dump/io/TestFunctionSerializer.java

```diff
@@ -0,0 +1,66 @@
+  /*
```

Comment:
instead of mock we can add a replication test for functions.

Reply:
done